cfgov/alerts/logging_handlers.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from alerts.github_alert import GithubAlert
+
+
+class CFGovErrorHandler(logging.Handler):
+    def __init__(self):
+        logging.Handler.__init__(self)
+
+    def emit(self, record):
+        GithubAlert({}).post(
+            title=record.message[:30],
```

Comment:
Truncating to 30 characters isn't ideal but I'm not sure how else to do this for now. Curious to see it in action before optimizing this.